### DISCUSSION TITLE
Sequential Jobs?

### DISCUSSION THREAD

**Discussion by gmbeas** [quartznet/quartznet#1293] [General]
is it possible to run a job after a job is finished?

Sequential?
Job1 -> Job2

**Comment by lahma** [comment]
You can always use the context to trigger a job after everything has been run.  There's also [JobChainingJobListener](https://github.com/quartznet/quartznet/blob/35b8532dc005af2fc907b6d2516c2e4bf40f833b/src/Quartz/Listener/JobChainingJobListener.cs) which might be of interest to you.

**Reply by gmbeas** [reply]
perfect, I will try the recommended and I will comment

**Reply by gmbeas** [reply]
Working!, thanks very veryyyyy veryyyyy much!
